code/game/objects/machinery/cic_maptable.dm

```diff
@@ -31,3 +31,8 @@
 /obj/machinery/cic_maptable/on_unset_interaction(mob/user)
 	. = ..()
 	user.client.screen -= map
+
+/obj/machinery/cic_maptable/droppod_maptable
+	name = "Athena Tactical Map"
+	desc = "A map that display a map of planetside, specialized in revealing potential areas to drop pod. This is especially useful to see where the frontlines and marines are at so that anyone droppodding can decide where to land. Pray that your land nav skills are robust to not get lost!"
```

Comment:
```suggestion
	desc = "A map that display the planetside AO, specialized in revealing potential areas to drop pod. This is especially useful to see where the frontlines and marines are at so that anyone droppodding can decide where to land. Pray that your land nav skills are robust to not get lost!"
```